### ISSUE TITLE
Sharing: fix `Warning: Failed propType` notice

### ISSUE THREAD

**Issue by jkudish** [Automattic/wp-calypso#1750]
Accessing the Sharing screen I get the following notice as a non-administrator:

```
Warning: Failed propType: Invalid prop `label` supplied to `SharingServiceExample`. Check the render method of `SharingServiceExamples`.
```

Initially reported in https://github.com/Automattic/wp-calypso/pull/1653#issuecomment-165265964

**Comment by aduth** [comment]
This warning is shown regardless of user role.

**Comment by aduth** [comment]
Fixed by #1653.
